docs/alert/index.html

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="author" content="Bharati Subramanian">
+        <meta name="description" content="elixir - documentation - alert component">
+        <link rel="shortcut icon" href="/logo-assets/favicon_io/favicon.ico" type="image/x-icon">
+
+        <title>elixir | alert</title>
+
+        <link rel="stylesheet" href="/styles.css">
+    </head>
+    <body>
+        <div class="grid-container">
+            <nav class="navbar flex--row flex--justify-end flex--align-center">
+                <div class="hamburger flex--col flex--justify-center flex--align-center">
+                    <button class="btn btn-icon btn-secondary">
+                        <span class="icon">
+                            <i class="fas fa-bars"></i>
+                        </span>
+                    </button>
+                </div>
+                <ul class="navlinks flex--row list-style-none flex--align-center">
+                    <li class="navlink">
+                        <a href="/" class="link text-link">
+                            Home
+                        </a>
+                    </li>
+                    <li class="navlink active">
+                        <a href="/docs" class="link text-link">
+                            Docs
+                        </a>
+                    </li>
+                    <li class="navlink">
+                        <a href="https://github.com/bharati-21/elixir-ui" target="_blank" class="link icon-link"><i class="fab fa-github"></i></a>
+                    </li>
+                </ul>
+                            
+            </nav>
+            <aside class="sidebar sidebar--close flex--col">
+                <div class="sidebar__header flex--row flex--align-center">
+                    <a href="/" class="brand flex--row flex--align-center flex--justify-center">
```

Comment:
I don't have much knowledge in naming convention but sometimes I see you used '--' or '__' or '-' so if it is the same naming convention then fine or you can try to change this.

Reply:
Yes, the same naming convention. It's called BEM. Block element modifier. 
E.g. sidebar is the container and the block header resides within the sidebar. hence `sidebar__header`. And as for --close, it refers to a changed state for the sidebar, which is the modifier.

Hope this helps.